generators/cypress/templates/src/test/javascript/cypress/support/keycloak-oauth2.ts.ejs

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable @typescript-eslint/no-namespace */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+// eslint-disable-next-line spaced-comment
+/// <reference types="cypress" />
+
+import { getAuthCodeFromLocation } from './utils';
+
+Cypress.Commands.add("kcLogin", (user: string) => {
```

Comment:
I am also not a big fan of prefixing with `kc`: I would be more explicit and use `keycloackLogin` or `loginWithKeycloak`